src/conductor/input_parser.star

```diff
@@ -32,7 +37,7 @@ def parse(
         + ": {}",
     )
 
-    # We filter the None values so that we can merge dicts easily
+    # Merge with defaults first
```

Comment:
I mean in the interest of keeping this PR small I would refrain from changing random comment lines :)